src/Microsoft.SqlTools.ServiceLayer/Microsoft.SqlTools.ServiceLayer.csproj

```diff
@@ -22,6 +22,7 @@
 	<Choose>
 		<When Condition="'$(BUILD_DOTNET_TOOL)' == 'true'">
 			<PropertyGroup>
+				<PackageVersion>1.0.0</PackageVersion>
```

Comment:
Sigh, reminded yet again that we still don't have the assembly versions match the zip version. This will do for now but we should really get that fixed at some point (and then use the same version here)